tests/integration_tests/security/test_jail.py

```diff
@@ -107,6 +92,8 @@ def test_exec_file_not_exist(test_microvm_with_api, tmp_path):
 
     # Error case 1: No such file exists
     pseudo_exec_file_path = tmp_path / "pseudo_firecracker_exec_file"
+    fc_dir = Path("/srv/jailer") / pseudo_exec_file_path.name / test_microvm.id
```

Comment:
nit: can probably make "/srv/jailer" a constant in defs.py

Reply:
I think I will leave it for now, but I will think about it.